### PR TITLE
5.0Doc: Translate/Rewrite ssl.md

### PR DESCRIPTION
**Pull request by hjianbo**

### CODE REVIEW

File: en_US/security/ssl.md [hunk 1]
```diff
@@ -66,146 +75,55 @@ File with psk identities and keys (`data/init.psk`) should be created:
...
-
-### `reuse_sessions`
-
-Boolean field, specifying whether to reuse SSL sessions. The default is `true`.
-
-### `depth`
-
-Integer field, maximum number of non-self-issued intermediate certificates that can follow the peer certificate in a valid certification path. The default is `10`.
+authentication {
+  url = "http://127.0.0.1:8080"
```
Comment (savonarola) [inline en_US/security/ssl.md]: `https://...`?

**Review by savonarola — DISMISSED** [review]

File: en_US/security/ssl.md [hunk 2]
```diff
@@ -21,22 +23,29 @@ listeners.ssl.default {
...
-  ssl.versions = ["tlsv1.3", "tlsv1.2", "tlsv1.1", "tlsv1"]
+  ssl_options.versions = ["tlsv1.3", "tlsv1.2", "tlsv1.1", "tlsv1"]
   tcp.backlog = 1024
   tcp.buffer = 4KB
 }
 ```
 
-Note that the `key.pem`,` cert.pem`, and `cacert.pem` under the default directory of `etc/certs` are self-signed certificates generated by EMQX Broker. Therefore, when testing with a client that supports TLS, you need to configure the above CA certificate `etc/certs/cacert.pem` to the client. For production use, securely issued certificates must be used.
+::: tip
+the `key.pem`,` cert.pem`, and `cacert.pem` under the default directory of `etc/certs` are self-signed certificates generated by EMQX Broker. Therefore, when testing with a client that supports TLS, you need to configure the above CA certificate `etc/certs/cacert.pem` to the client.
```
Comment (k32) [inline en_US/security/ssl.md]: Capitalize t

File: en_US/security/ssl.md [hunk 3]
```diff
@@ -66,146 +75,55 @@ File with psk identities and keys (`data/init.psk`) should be created:
...
-### `ciphers`
+- When **TLS Verify** is enabled, the **CA Cert** field must be filled in to verify the legitimacy of the server certificate.
+- When the server needs to verify the client certificate, then **TLS Cert** and **TLS Key** must be filled in.
+- **SNI** meanings Server Name Indication, indicates whether the server domain name and certificate are verified to be the same; A null value means no validation.
 
-Optional list of ciphers to use. The default list is:
+In addition, it is also can be configured in the `emqx.conf`:
 
 ```
-# TLS 1.3:
```
Comment (zmstone) [inline en_US/security/ssl.md:134]: we need these ciphers documented ?

**Review by zmstone — APPROVED** [review]

File: en_US/security/ssl.md (hunk 3 above)
Comment (hjianbo) [reply en_US/security/ssl.md:134]: I think so.  Can I update it in another PR?

**Pull request merged**